src/App.js

```diff
@@ -21,6 +23,11 @@ function App() {
       </Switch>
     </div>
   );
-}
+};
 
-export default App;
+export default geolocated({
+  positionOptions: {
```

Comment:
can u explain what this does?

Reply:
This sets a decision time out on users choice to share location or not

Reply:
But you don’t use it. You use JS to get the position of the user. This package is not used

Reply:
fixed